web3.js/src/loader.js

```diff
@@ -169,22 +169,15 @@ export class Loader {
       });
       transactions.push(
         sendAndConfirmTransaction(connection, transaction, [payer, program], {
-          commitment: 'single',
+          commitment: 'singleGossip',
           skipPreflight: true,
         }),
       );
 
       // Delay between sends in an attempt to reduce rate limit errors
-      const REQUESTS_PER_SECOND = 4;
-      await sleep(1000 / REQUESTS_PER_SECOND);
-
-      // Run up to 8 Loads in parallel to prevent too many parallel transactions from
-      // getting retried due to AccountInUse errors.
-      //
-      // TODO: 8 was selected empirically and should probably be revisited
-      if (transactions.length === 8) {
-        await Promise.all(transactions);
-        transactions = [];
+      if (connection._rpcEndpoint.includes('solana.com')) {
```

Comment:
🙈 - looking forward to a more dynamic 429-response based approach :)

Reply:
Will get to it soon, here's an issue: https://github.com/solana-labs/solana/issues/13061